src/Result/ResultHandler.php

```diff
@@ -34,46 +34,36 @@
  *
```

Comment:
Did we delete `OrderConfirmationTest`?

Reply:
Yes, as OrderConfirm will not be present. The test are now under `Nosto\Test\Unit\Result\GraphqlOrderTest`